lib/recurly/http.rb

```diff
@@ -13,6 +13,7 @@ def initialize(resp, request)
         @rate_limit = resp["x-ratelimit-limit"].to_i
         @rate_limit_remaining = resp["x-ratelimit-remaining"].to_i
         @rate_limit_reset = Time.at(resp["x-ratelimit-reset"].to_i).to_datetime
+        @total_records = resp["recurly-total-records"].to_i
```

Comment:
We may want to put this in a conditional or else it will come out as `0`. perhaps:
```ruby
@total_records = resp["recurly-total-records"]&.to_i
```

Reply:
Updated